.scripts/update_readme_badges.sh

```diff
@@ -54,6 +54,10 @@ for BUILD_WORKFLOW_FILE in .github/workflows/build_*.yaml; do
     echo >> "$BADGES_TMP"
   fi
 done
+# This needs to add the remaning empty columns of the last row in the table
+# This is a hack to fix the status quo and make markdownlint happy.
+echo -n "| | | |" >> "$BADGES_TMP"
```

Comment:
Hmm, I though I'd solved that when first introduced.

I'll take a look again. It shouldn't be too hard to calculate the remaining cells.